src/components/MoneyRequestHeader.tsx

```diff
@@ -61,7 +61,8 @@ function MoneyRequestHeader({report, parentReportAction, policy, shouldUseNarrow
     const {translate} = useLocalize();
     const [isDeleteModalVisible, setIsDeleteModalVisible] = useState(false);
     const [shouldShowHoldMenu, setShouldShowHoldMenu] = useState(false);
-    const moneyRequestReport = parentReport;
+    const isSelfDMTrack = ReportUtils.isTrackExpenseReport(report) && ReportUtils.isSelfDM(parentReport);
```

Comment:
NAB: variable clarity 

```suggestion
    const isSelfDMTrackExpenseReport = ReportUtils.isTrackExpenseReport(report) && ReportUtils.isSelfDM(parentReport);
```

Reply:
Updated